redirects-list.json

```
@@ -1,5 +1,11 @@
 [
-  "^https://www\\.cml\\.dev(.*)?                                                          https://cml.dev$1",
-  "^/(?:docs|documentation)(/.*)?$                                                        /doc$1",
-  "^/(.+)/$                                                                               /$1"
+  "^https://(?:www\\.)?cml\\.dev(.*)?                                      https://cml.dev$1",
```

why do we change this record?

---

surely we want suppression of `www` just like https://github.com/iterative/dvc.org/blob/88bc65300c3372df0b32ae826869e58ffbf3ef28/redirects-list.json#L2 & https://github.com/iterative/dvc.org/blob/88bc65300c3372df0b32ae826869e58ffbf3ef28/redirects-list.json#L15

---

oh wait nvm. Not really necessary. Effectively identical.

```suggestion
  "^https://www\\.cml\\.dev(.*)?                                           https://cml.dev$1",
```